ci/jenkins/test.sh

```diff
@@ -541,7 +547,11 @@ function deliver_antrea {
     fi
     chmod -R g-w build/images/ovs
     chmod -R g-w build/images/base
-    DOCKER_REGISTRY="${DOCKER_REGISTRY}" ./hack/build-antrea-linux-all.sh --pull
+    if [[ $TESTBED_TYPE == "kind" ]]; then
```

Comment:
I don't think the `if` statement adds value. Even if the testbed type is not `"kind"`, we could still honor the provided custom build tag.

Reply:
actually, we only support this for kind as of now, if we use this in other testbeds, then we will have to update the yamls also for those testbeds.

Reply:
yes, you should always call `./hack/generate-manifest.sh` regardless of the testbed. The test mode determines which flags you pass to the script.

Reply:
We can easily address this. We just need to set `IMG_TAG` when `BUILD_TAG` is provided, and invoke `generate-manifest.sh`, regardless of the testbed.